### PR TITLE
Utils change

### PR DESCRIPTION
**Pull request by doronp**

### CODE REVIEW

File: pkg/multiTenancyPlugins/utils/utils.go [hunk 1]
```diff
@@ -92,6 +92,45 @@ func RandStringBytesRmndr(n int) string {
 	return string(b)
 }
 
+type commandEnum string
```
Comment (doronp) [inline pkg/multiTenancyPlugins/utils/utils.go]: Do we like the name? Are there missing commands? @KennethNagin @andreyod ?

File: pkg/multiTenancyPlugins/utils/utils.go [hunk 2]
```diff
@@ -92,57 +92,43 @@ func RandStringBytesRmndr(n int) string {
...
-		return "container" + paramsArr2[2]
-	}
-	if len(paramsArr1) == 2 {
+		if len(networksParams) > 0 {
+			return "networkdelete"
+		}
 
-		if paramsArr1[1] == "json" {
-			return "listContainers"
+	case "GET", "POST":
```
Comment (doronp) [inline pkg/multiTenancyPlugins/utils/utils.go:199]: The code becomes more generic and short but more complex. I think with the printings above it is OK to maintain.

File: pkg/multiTenancyPlugins/utils/utils.go [hunk 3]
```diff
@@ -92,57 +92,43 @@ func RandStringBytesRmndr(n int) string {
...
 
-	paramsArr1 := containers.FindStringSubmatch(r.URL.Path)
-	paramsArr2 := containersWithIdentifier.FindStringSubmatch(r.URL.Path)
-	networksParams := networks.FindStringSubmatch(r.URL.Path)
-	//assert the it is not possible for two of them to co-Exist
-
-	log.Debug(paramsArr1)
-	log.Debug(paramsArr2)
-
+	log.Debug(containersParams)
```
Comment (doronp) [inline pkg/multiTenancyPlugins/utils/utils.go:186]: Used to ease the debug on the parsing

File: pkg/multiTenancyPlugins/utils/utils.go [hunk 4]
```diff
@@ -92,6 +92,45 @@ func RandStringBytesRmndr(n int) string {
...
+	CONTAINER_EXPORT  commandEnum = "containerExport"
+	CONTAINER_CHANGES commandEnum = "containerChanges"
+	CONTAINER_JSON    commandEnum = "containerJson"
+	CONTAINER_TOP     commandEnum = "containerTop"
+	CONTAINER_LOGS    commandEnum = "containerLogs"
+	CONTAINER_STATS   commandEnum = "containerStats"
+
+	//SKIP ...
+	//POST
+	CONTAINER_KILL    commandEnum = "containerKill"
```
Comment (doronp) [inline pkg/multiTenancyPlugins/utils/utils.go]: Missing container create here , tnx @tsofnat

**Comment by ezrasilvera** [comment]
I'm worried about parsing that is based on the number of arguments and also hardcode these numbers into the conditions .. E,gg.,  return "containers" + containersParams[3] //S
Also it seems risky that the returned value is taken automatically from the command, we should explicitly return the enum you defined. 
The current approach is indeed shorter but this dependency is risky

File: pkg/multiTenancyPlugins/utils/utils.go [hunk 5]
```diff
@@ -92,57 +92,85 @@ func RandStringBytesRmndr(n int) string {
...
-	if len(paramsArr1) == 2 {
+		if len(networksParams) > 0 {
+			return "networkdelete"
+		}
 
-		if paramsArr1[1] == "json" {
-			return "listContainers"
+	case "GET", "POST":
+		if len(containersParams) == 4 && containersParams[2] != "" {
+			return "container" + containersParams[2]
```
Comment (ezrasilvera) [inline pkg/multiTenancyPlugins/utils/utils.go:201]: The enum is defined with capital letter (e.g., containerStart) , I think that the argument in the http request is with lower case ...

**Comment by KennethNagin** [comment]
If I understand you have removed the reading of the apifilter.json file and the generation of the apiDisableMap.  Please explain how different deployments will now dynamically disable implemented api requests without recompiling.

**Comment by ezrasilvera** [comment]
@KennethNagin why did you close this PR ?   You probably just wanted to comment, right ?

**Comment by ezrasilvera** [comment]
@doronp Let me clarify @KennethNagin comment,  The idea was to be able to easily disable functions that are implemented (obviously not implemented functions are not supported) , in such a away that we could have different functions disabled on different deployments. 
Two options I see
1) Have an external user readable file that can be edited and is loaded by the manager
2) Build a different image for each deployment 
If any f you see different approach please share....

**Comment by ezrasilvera** [comment]
@doronp you should rebase your code to multiTennat and push into this branch to update the PR , we can't merge it ...

**Comment by KennethNagin** [comment]
The updates related to the command name made in apifilter does not seem to be consistent with the command enumerations, e.g.  listNetwork is use and it should be networkList.

**Pull request merged**